src/store/modules/wallet.js

```diff
@@ -217,9 +281,88 @@ const actions = {
         from: address,
       };
       await this.$api.post(postUserV2Login(), data);
-      commit(types.WALLET_SET_LOGIN_ADDRESS, address);
+      await dispatch('walletFetchSessionUserInfo', address);
+    } catch (error) {
+      commit(WALLET_SET_USER_INFO, null);
+      throw error;
+    } finally {
+      commit(WALLET_SET_IS_LOGGING_IN, false);
+    }
+  },
+
+  async walletLogout({ commit }) {
+    try {
+      commit(WALLET_SET_USER_INFO, null);
+      commit(WALLET_SET_FOLLOWERS, []);
+      await this.$api.post(postUserV2Logout());
+    } catch (error) {
+      throw error;
+    }
+  },
+  async walletUpdateEmail({ state, commit }, email) {
+    try {
+      await this.$api.$post(
+        apiUserV2WalletEmail({ wallet: state.loginAddress, email })
+      );
+      commit(WALLET_SET_USER_INFO, { emailUnconfirmed: email });
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.error(error);
+      throw error;
+    }
+  },
+  async walletVerifyEmail({ state, commit }, token) {
+    try {
+      await this.$api.$put(
+        apiUserV2WalletEmail({ wallet: state.loginAddress, token })
+      );
+      commit(WALLET_SET_USER_INFO, {
+        email: state.emailUnconfirmed,
+        emailUnconfirmed: '',
+      });
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.error(error);
+      throw error;
+    }
+  },
+  async fetchFollowers({ commit, dispatch }, address) {
+    try {
+      const { followers } = await this.$axios.$get(getUserFollowers(address));
+      commit(WALLET_SET_FOLLOWERS, followers);
+      if (followers.length) {
+        dispatch('lazyGetUserInfoByAddresses', followers);
+      }
+    } catch (error) {
+      throw error;
+    }
+  },
+  async walletCreatorFollow({ state, commit }, creator) {
```

Comment:
Very strange name, can we just use`followCreator` ?

Reply:
Wallet is the namespace